com.unity.ml-agents/Tests/Editor/PublicAPI/Unity.ML-Agents.Editor.Tests.PublicAPI.asmdef

```diff
@@ -0,0 +1,27 @@
+{
+    "name": "Unity.ML-Agents.Editor.Tests.PublicAPI",
```

Comment:
I just copied this from `Unity.ML-Agents.Editor.Tests` and changed the name - anything else I should update?